app/sagas/createChannel.js

```diff
@@ -67,14 +71,17 @@ const handleRequest = function* handleRequest({ data }) {
 				encrypted
 			});
 			sub = yield call(createChannel, data);
+			if (sub.teamId) {
```

Comment:
Why is this inside `channel` creating logic instead of `team`'s?

Reply:
Because `groups.create` and `channels.create` now can return a `teamId`, I'm using this validation here so *if* we receive a `teamId` then we will then add the room to the team with `addRoomsToTeam`